js/copy-to-llms.js

```diff
@@ -459,6 +465,12 @@
 
         // Each dropdown option maps to one of the shared helpers or a new-tab prompt.
         switch (action) {
+          case 'view-markdown': {
+            trackButtonClick('view_page_markdown');
+            const mdUrl = getMarkdownUrl(slug);
+            const opened = window.open(mdUrl, '_blank', 'noopener');
```

Comment:
```suggestion
            window.open(mdUrl, '_blank', 'noopener');
```

Since we don't use the variable, we don't need it